crates/relayer-types/src/applications/ics31_icq/response.rs

```diff
@@ -7,7 +7,7 @@ use prost::Message;
 use std::prelude::v1::*;
 use std::vec;
 use tendermint::merkle::proof::ProofOps as TendermintProofOps;
-use tendermint_proto::crypto::{ProofOp, ProofOps};
+use tendermint_proto::v0_34::crypto::{ProofOp, ProofOps};
```

Comment:
Could you check if this is needed? Were there changes in v0.37?

Reply:
No changes as far as I can tell. This change has been reverted now, and the import path is currently pointing at the `v0_37` protobuf types.